src/main/java/com/mate/model/ShoppingCart.java

```diff
@@ -0,0 +1,51 @@
+package com.mate.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+
+@Entity
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    @MapsId
+    @OneToOne
```

Comment:
Maybe it would be better to add annotation JoinColumn(name = "some_id")) here to avoid duplication of columns id?

Reply:
My columns not duplicated

Reply:
Already tested too, it works. Sorry, my fault. =)